README.md

```diff
@@ -1,3 +1,25 @@
+## Requirements
+
+* scrapy>=2.5.1
+* zyte-api>=0.1.2
+* twisted>=21.7.0
+
+If you are deploying on Scrapy Cloud, then add the below packages to your requirements.txt file.
+
+* zyte-api>=0.1.2
+* twisted>=21.7.0
+* git+https://github.com/scrapy-plugins/scrapy-zyte-api.git
+
+## Installation
+
+It is not yet available on PyPI. However, it can be installed directly from GitHub:
```

Comment:
This would be on us. 😅 I'll do this one soon.

Reply:
That's fine. 👍 😄  I think scrapy-crawlera-fetch https://github.com/scrapy-plugins/scrapy-crawlera-fetch is also not available on the pip yet.

Reply:
Great thanks 😄 👍